File: packages/gizmos-env/lib/energy_pool.ts
import type { Energy } from './common'
import { shuffle } from './utils'

const ENERGY_POOL: Energy[] = [
  ...new Array(13).fill('red'),
  ...new Array(13).fill('black'),
  ...new Array(13).fill('blue'),
  ...new Array(13).fill('yellow'),
]
export function init_energy_pool() {
  return shuffle(ENERGY_POOL)
}

File: py/energy_pool.py
from random import shuffle

from common import Energy

ENERGY_POOL: list[Energy] = ['red']*13 + \
    ['yellow']*13 + ['blue']*13 + ['black']*13


def init_energy_pool() -> list[Energy]:
    E = list(ENERGY_POOL)
    shuffle(E)
    return E
